Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent none
+    stages {
+        stage('Parallel Stage') {
+            when {
+                branch 'master'
+            }
+            failFast true
+            parallel {
+                stage('Branch A') {
+                    agent {
+                        label "linux"
+                    }
+                    steps {
+                        sh 'mvn -B clean verify'
```

Comment:
A nicer script would set `-Dmaven.test.failure.ignore` (IIRC) and use `junit` but this is good enough for now.